frontend/src/pages/modelRegistry/screens/RegisteredModels/RegisteredModelsTableColumns.ts

```diff
@@ -20,7 +20,7 @@ export const rmColumns: SortableData<RegisteredModel>[] = [
     sortable: (a: RegisteredModel, b: RegisteredModel): number => {
       const first = parseInt(a.lastUpdateTimeSinceEpoch);
       const second = parseInt(b.lastUpdateTimeSinceEpoch);
-      return new Date(first).getTime() - new Date(second).getTime();
+      return new Date(second).getTime() - new Date(first).getTime();
```

Comment:
Hmm.. ideally we would revert this and pass a `defaultSortDirection="desc"`, since it's a little confusing to have the ascending sort of last update time be newest to oldest... but apparently there is no `defaultSortDirection` prop on our Table or on the `useTableColumnSort` hook it uses... probably out of scope to fix that here.

I suppose it makes some sense to have this sort flipped because the values are displayed as "time ago" instead of a timestamp, so in fact the shown values are sorted ascending when they are newest to oldest. So yeah let's leave it as you have it.